README.md

```diff
@@ -1,61 +1,93 @@
-Dogtag PKI
-==========
+# Dogtag PKI
 
-[![Build Status](https://travis-ci.org/dogtagpki/pki-nightly-test.svg?branch=master)](https://travis-ci.org/dogtagpki/pki-nightly-test)
+![Required Tests](https://github.com/dogtagpki/pki/workflows/Required%20Tests/badge.svg)
+![QE Tests](https://github.com/dogtagpki/pki/workflows/QE%20Tests/badge.svg)
 
-(C) 2008 Red Hat, Inc.
-All rights reserved.
+The Dogtag Certificate System is an enterprise-class open source Certificate Authority (CA). It is a full-featured system, and has been hardened by real-world deployments. It supports all aspects of certificate lifecycle management, including key archival, OCSP and smartcard management, and much more.
 
-This Certificate System is open-source software.
+There are 5 different subsystems included in the Dogtag PKI suite:
 
-Please comply with the LICENSE contained in each of
-the individual components, and the EXPORT CONTROL
-regulations defined at:
+1. [Certificate Authority (CA) subsystem](https://www.dogtagpki.org/wiki/Certificate_Authority)
+2. [Key Recovery Authority (KRA) subsystem](https://www.dogtagpki.org/wiki/Key_Recovery_Authority)
+3. [Online Certificate Status Protocol (OCSP) subsystem](https://www.dogtagpki.org/wiki/OCSP_Manager)
+4. [Token Key Service (TKS) subsystem](https://www.dogtagpki.org/wiki/Token_Key_Service)
+5. [Token Processing System (TPS) subsystem](https://www.dogtagpki.org/wiki/Token_Processing_System)
 
-http://www.dogtagpki.org/wiki/PKI_Download
+## Documentation
 
-## Content
+The best place to start learning about the product is the [Dogtag PKI Wiki](https://www.dogtagpki.org)
 
-These directories contain the following:
+## Installing
 
-* base
+### Fedora/CentOS/RHEL
 
-  Contains most of the base source code
-  needed to build this project.  Note that
-  this directory does NOT contain
-  implementation specific user-interface
-  components required to build a working
-  Certificate System.
+To install the **whole Dogtag PKI suite**:
 
-* cmake
+````bash
+sudo dnf install dogtag-pki
+````
 
-  These files and this directory contain
-  the top-level files necessary to integrate
-  the CMake build system in pki.
+To install **individual subsystems**:
 
-* scripts
+````bash
+sudo dnf install pki-ca pki-kra pki-ocsp pki-tks pki-tps
+````
 
-  Contains "scripts" used by this
-  certificate system.  This directory
-  contains numerous "compose" scripts
-  useful for building RPMS/SRPMS of the
-  various certificate system components.
+To install **web UI theme packages**:
 
-* themes
+````bash
+sudo dnf install dogtag-pki-server-theme dogtag-pki-console-theme
+````
 
-  Contains the scripts and user-interface
-  components to customize PKI web UI and
-  console.
+## Deploying
 
-* tools
+After successful installation of the packages, follow the below steps to deploy intended subsystems:
 
-  Contains utilities useful to
-  certificate system components.
+- [Deploy CA](docs/installation/ca/Installing_CA.md)
```

Comment:
We can add a link to acme/Installing_PKI_ACME_Responder.md as well.

Reply:
Done